semgrep_output_v1.atd

```diff
@@ -241,7 +241,7 @@ type core_match_extra <python decorator="dataclass(frozen=True)"> = {
    * Added in semgrep 1.37.0 *)
   ?validation_state : validation_state option;
   (* EXPERIMENTAL: added by secrets post-processing & historical scanning code.
-   * Since 1.60.0. *)
+   * Since 1.62.0. *)
```

Comment:
I assumed this comment was attached to validation_state?

Reply:
This is for `historical_info`. We haven't sent it yet so I would just be updating this comment here.